src/dialects/mssql/query/compiler.js

```diff
@@ -75,6 +76,12 @@ assign(QueryCompiler_MSSQL.prototype, {
         sql = '';
       }
     }
+
+    if(queryContext && queryContext.identityInsert) {
```

Comment:
So this leaves connection to a state it was in the last time and doesn't reset it to on, after query is done? Looks like a bug.

Reply:
@elhigu Hmmm, no its not a bug, connection/session state gets reset afterwards with 
```
sql = `SET IDENTITY_INSERT ${this.tableName} ON; ${sql}; `;
sql += `SET IDENTITY_INSERT ${this.tableName} OFF;`;
```